### PR TITLE
ID computer can now assign planetside jobs, command secretary, IAA

### PR DESCRIPTION
**Pull request by Atermonera**
Also cleans up that template with real indentation and adds some closing {{/if}} tags.
Captain and Custom need to stay separate within the assignment menu, because I don't think there's a particularly clean way to do the custom thing.
Centcom jobs lose a touch of special formatting that I didn't even know they have because who even uses ID computers with centcom access

Edit: IAA and command secretary also now available. Command secretaries have keycard auth access, per instruction from superiors.

### CODE REVIEW

**Comment by Atermonera** [comment]
Oh, right, and 
Tested

**Comment by Belsima** [comment]
I've noticed IAA isn't in that list; is that intentional or?

**Comment by Atermonera** [comment]
They aren't listed under any of the lists of positions, and looking at it, command secretary also isn't on there because it's only listed under command_positions, which I explicitly don't use. I'll look into it later

**Comment by Atermonera** [comment]
Testing (Compiled on NS, so planetside stuff doesn't show up):
[Selection](https://puu.sh/zcFbz/9153b44948.png)
[IAA Access](https://puu.sh/zcEX7/8dc9c146f3.png)
[Secretary Access](https://puu.sh/zcFiV/59d9d52dd4.png)

File: html/changelogs/Atermonera - id_comp.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: Atermonera
+
+delete-after: True
+changes:
+  - tweak: "ID computer can set command secretary and IAA access"
+  - tweak: "Command secretary has keycard auth. access"
```
Comment (Anewbe) [inline html/changelogs/Atermonera - id_comp.yml]: This is only true if the config for minimum access is on? Otherwise they don't?

**Comment by Anewbe** [comment]
Is this complete, Ater?

**Comment by Atermonera** [comment]
Yes

**Pull request merged**